docs/api/session.md

```diff
@@ -56,7 +57,11 @@ be thrown if an empty string is provided.
 
 To create a `Session` with `options`, you have to ensure the `Session` with the
 `path` has never been used before. There is no way to change the `options`
-of an existing `Session` object.
+of an existing `Session` object. The optional `quota` parameter can be used to
+specify the a quota size that can be used by the session, which will be returned as the quota provided by navigator.storage.estimate(). If the `quota` is not
+provided, the size of the volume on which path is located will be used. A smaller
+quota size will allow the LRU algorithm to evict more data from the cache, though there
```

Comment:
Many people won't know what an LRU algorithm is - perhaps link to an explainer?

Reply:
Fixed.